app/decorators/service_provider_session_decorator.rb

```diff
@@ -30,7 +30,11 @@ def generate_custom_alert(section, args)
   end
 
   def sp_logo
-    sp.logo || DEFAULT_LOGO
+    if sp.logo.present?
+      sp.logo
+    else
+      DEFAULT_LOGO
+    end
```

Comment:
I think this can be simplified a tad

```suggestion
    sp.logo.presence || DEFAULT_LOGO
```

Reply:
Rubocop agrees: `app/decorators/service_provider_session_decorator.rb:33:5: C: Rails/Presence: Use sp.logo.presence || DEFAULT_LOGO instead of if sp.logo.present?` 😛